mvlearn/datasets/base.py

```diff
@@ -53,9 +53,8 @@ def load_UCImultifeature(select_labeled="all", views="all", shuffle=False,
 
     References
     ----------
-    .. [#1Data] M. van Breukelen, R.P.W. Duin, D.M.J. Tax, and J.E. den Hartog,
```

Comment:
Are we not still going to keep this reference, since this is the original data?

Reply:
saw it on the UCI as a relevant paper, didn't realize was the original. Cited both UCI and this one.